Gruntfile.js

```diff
@@ -70,9 +70,16 @@ module.exports = function(grunt) {
             }
         },
 
-        jsFiles: [
+        jsHeadFiles: [
             // Include:
+            'source/js/head-scripts.js'
+        ],
+
+        jsFootFiles: [
+            // Include:
+            'source/js/jquery-2.0.0b2.js',
```

Comment:
Do you think its better to include the jQuery here or just pull it from the jQuery CDN?